modules/nodes-cluster-admission-configuring.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-containers-controllers.adoc
+
+[id="nodes-containers-controllers-configuring_{context}"]
+= Configuring admission controllers
+
+
+
+.Procedure
+
+
+. Edit the `kubeapiserver` to add an `admissionPluginConfig`:
++
+----
+oc edit kubeapiserver
+----
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1 <1>
+kind: KubeAPIServer
+
+....
+
+unsupportedConfigOverrides:
+  admissionPluginConfig:
+    enabledPlugins:
+    - name: <plugin-name> <2>
+      <configurations>
+----
+<1> Specify the API to use: `operator.openshift.io/v1` or `autoscaling.openshift.io`, 
+as indicated in *Understanding admission controllers*.
+<2> Add the name of the plugin and any required configuration, as indicated in *Understanding admission controllers*..
++
+For example:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1 <1>
+kind: KubeAPIServer
+
+unsupportedConfigOverrides:
+  admissionPluginConfig:
+    enabledPlugins:
+    - DefaultTolerationSeconds <2>
+----
++
+[source,yaml]
+----
+apiVersion: autoscaling.openshift.io
+kind: KubeAPIServer
+
+....
+
+unsupportedConfigOverrides:
+  admissionPluginConfig:
+    enabledPlugins:
+    - name: RunOnceDuration
+----
+
```

Comment:
@sunilcio @ravisantoshgudimetla How do you handle different APIs? Create multiple kubeapiserver objects with different names?